packages/client/hmi-client/src/components/widgets/slider-panel.vue

```diff
@@ -11,7 +11,11 @@
 					:class="`slider-header-item pi ${directionMap[direction].iconOpen}`"
 					@click="emit('update:isOpen', false)"
 				/>
-				<h4 class="slider-header-item">{{ header }}</h4>
+				<slot name="header"></slot>
```

Comment:
Why do you need this here?

Reply:
This is the dropdown button that is provided from the ..header.. file
This allows us to have the `CART` the `dropdown` and the `>>` on the same line.

Previously the header div containing what is now `CART` and `>>` wouldnt contain the drop down button so I could not make them all level with each other

Reply:
I can use v-slot instead to make it more consistent with what we have